modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/EditCTCollectionMVCRenderCommand.java

```diff
@@ -51,6 +56,24 @@ public String render(
 		long ctCollectionId = ParamUtil.getLong(
 			renderRequest, "ctCollectionId");
 
+		ThemeDisplay themeDisplay = (ThemeDisplay)renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		try {
+			if ((ctCollectionId != 0) &&
+				!_ctCollectionModelResourcePermission.contains(
+					themeDisplay.getPermissionChecker(), ctCollectionId,
+					ActionKeys.VIEW)) {
+
+				throw new PortalException();
```

Comment:
Use `_ctCollectionModelResourcePermission.check` so that we don't need to manually throw the exceptions.